lib/screens/home_screen.dart

```diff
@@ -159,6 +159,37 @@ class HomeScreen extends StatelessWidget {
               ),
             );
           }),
+        drawer: Drawer(
+                child: ListView(
+                  // Important: Remove any padding from the ListView.
```

Comment:
if its something that needs to be done, create a ToDo task in project board for it